documentation/source/library_reference.rst

```diff
@@ -141,6 +141,9 @@ Clock
 .. autoclass:: cocotb.clock.Clock
     :members:
     :member-order: bysource
+    :noindex:
+..
+   Primary occurrence in this document under "Interacting with the Simulator", so don't index this
```

Comment:
Might make more sense to remove one of these, documenting the same class twice on the same page seems confusing.

Reply:
Done.